CHANGELOG.md

```diff
@@ -1,19 +1,31 @@
 # dbt_jira v1.1.0
 
-## Schema Updates
-
-**6 total changes • 4 possible breaking changes**
-
-| Data Model                                    | Change Type | Old  | New                               | Notes                                                             |
-|---------------------------------------------------|-------------|----------|-------------------------------------------|-------------------------------------------------------------------|
+## Schema/Data Change
+**9 total changes • 7 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ---------- | ----------- | -------- | -------- | ----- |
+| `stg_jira__team` | New model | N/A | New staging model | Added staging model for Jira teams functionality. |
+| `jira__daily_issue_field_history` | Column addition | N/A | `team` | Added team column to track team assignments for issues over time. |
```

Comment:
I think with the suggested changes we can remove Row 9, as we are no longer bringing in `team` by default.